encrypted-media-respec.html

```diff
@@ -2115,13 +2139,12 @@
                   <p>
                     If this object's <var>session type</var> is <a def-id="persistent-usage-record-session"></a>, store the <var>record of key usage</var>.
                   </p>
+                  <p class="note">
+                    This step may be run asynchronously, including after the document has unloaded.
```

Comment:
We probably don't want any persistence operations to block the main thread. Thus, it might make sense to replace the first part of this note by insering "run the following steps in parallel:" after the comma at line 2140.